pkg/cmd/ssh/ssh_test.go

```diff
@@ -303,14 +315,16 @@ var _ = Describe("SSH Command", func() {
 
 	Describe("RunE", func() {
 		BeforeEach(func() {
+			seedClientConfig, err := clientcmd.NewClientConfigFromBytes(seedKubeconfigSecret.Data["kubeconfig"])
+			Expect(err).NotTo(HaveOccurred())
+			clientProvider.EXPECT().FromClientConfig(gomock.Eq(seedClientConfig)).Return(seedClient, nil).AnyTimes()
```

Comment:
I think `FromClientConfig` is called exactly once per test. If yes remove `AnyTimes`